buildtools/shrinkwrap/shrinkwrap.js

```diff
@@ -22,27 +22,28 @@ var _ = require('lodash'),
 
 module.exports = function (grunt) {
 
-  function removeResolved(json) {
+  function removeResolvedAndFrom(json) {
```

Comment:
I'll remove it; although, I didn't make any progress resolving the extraneous modules problem consistently. I ended up installing what I needed and running `npm run shrinkwrap` to get a reasonable shrinkwrap.